python/foglamp/common/storage_client/storage_client.py

```diff
@@ -532,3 +534,209 @@ def purge(cls, age=None, sent_id=0, size=None, flag=None):
             raise StorageServerError(code=r.status, reason=r.reason, error=jdoc)
 
         return jdoc
+
+
+class ReadingsStorageClientAsync(ReadingsStorageClient):
```

Comment:
this should extend `StorageClient` and to be specific `StorageClientAsync` when its in!